doc/classes/@GlobalScope.xml

```diff
@@ -105,7 +106,7 @@
 			<description>
 				Rounds [code]x[/code] upward (towards positive infinity), returning the smallest whole number that is not less than [code]x[/code].
 				[codeblock]
-				i = ceil(1.45)  # i is 2
+				var i = ceil(1.45)  # i is 2
 				i = ceil(1.001) # i is 2
```

Comment:
```suggestion
				i = ceil(1.001) # i is 2.0
```
Maybe for a different PR, but if `ceil` returns a float it should probably be `2.0`